spark/src/main/resources/python/zeppelin_pyspark.py

```diff
@@ -44,15 +49,59 @@ def flush(self):
 class PyZeppelinContext(dict):
   def __init__(self, zc):
     self.z = zc
+    self.max_result = 1000
```

Comment:
make this from a interpreter property? I think there is a PR on something like this